projects/net-snmp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google Inc.
```

Comment:
We don't update copyright years when modifying a file. Only new ones.

Reply:
Thanks, fixed.